tunix/rl/agentic/rewards/reward.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, Callable
+from tunix.rl.agentic.rewards.reward_types import RewardOutput
+
+# ---------- ① Registry ----------
+_REGISTRY: Dict[str, Callable[[Dict, str], RewardOutput]] = {}
+
+def register(name: str):
```

Comment:
I think it's better to add an unregister method, so later it can be used by unit tests